Documentation/contributing/release/index.rst

```diff
@@ -19,9 +19,6 @@ upcoming stable releases.
 
    organization
    backports
-   stable
-   rc
-   feature
```

Comment:
Please update the paragraph above (`This includes information around the release cycles and guides for developers responsible for [...] preparing upcoming stable releases`, and even ). As suggested by David, a pointer to the `release` repo would be helpful.

Reply:
Ups, good point!